spec/rest-middleware.spec.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+describe("restMiddleware", () => {
+    var configMock, helmetMock, logger, restMiddleware, serverMock, restifyLinks;
+
+    beforeEach(() => {
+        var LoggerMock, RestMiddleware;
+
+        RestMiddleware = require("../lib/rest-middleware");
+        LoggerMock = require("./mock/logger-mock");
+        helmetMock = jasmine.createSpyObj("helmetMock", [
+            "frameguard",
+            "hidePoweredBy",
+            "hsts",
+            "ieNoOpen",
+            "noCache",
+            "noSniff",
+            "xssFilter"
+        ]);
+        serverMock = jasmine.createSpyObj("serverMock", [
+            "use"
+        ]);
+        configMock = {
+            baseUrl: "http://localhost:8443",
+            https: false
+        };
+        restifyLinks = jasmine.createSpy();
+        logger = new LoggerMock();
+        restMiddleware = new RestMiddleware(helmetMock, logger, restifyLinks);
+    });
+
+    function expectNormalMiddlewareWasCalled () {
```

Comment:
Please document this function, move it just below the `var` line to help conceptually simulate the function hoisting that JavaScript performs and then eliminate that one space before the open parenthesis.